.github/workflows/ubuntu.yml

```diff
@@ -101,7 +101,7 @@ jobs:
 
   build_3D_sp:
     name: GCC 3D & RZ w/ MPI, single precision
-    runs-on: ubuntu-22.04
```

Comment:
I admittedly had intentionally a few 22.04 runners because it will cover older g++ and clang++ compilers that way.

Reply:
Please revert this one.

Reply:
Done in [dac25b8](https://github.com/ECP-WarpX/WarpX/pull/5731/commits/dac25b8bd3e976914114abee75f068d2be40df6d).